server/aesdsocket.c

```diff
@@ -317,15 +355,16 @@ int main(int argc, char *argv[])
     addr_size = sizeof their_addr;
 
 	
-    socket_file_fd = open(FILE_PATH, O_RDWR | O_CREAT | O_APPEND, 0666);
-
+    socket_file_fd = open(file_path, O_RDWR | O_CREAT | O_APPEND, 0666);	/////
```

Comment:
@anuh7 this is going to cause the driver unload to fail I believe.  Also you won't want to O_CREATE when opening on your driver endpoint, you should be creating the device node and connecting to your driver in a startup script.